plugins/inputs/sflow/decoder_test.go

```diff
@@ -479,6 +524,8 @@ func TestIPv6SW(t *testing.T) {
 				"ether_type":       "IPv6",
 				"header_protocol":  "ETHERNET-ISO88023",
 				"input_ifindex":    "257",
+				"ip_dscp":          "0",
+				"ip_ecn":           "0",
```

Comment:
These should remain fields, possibly in the future we could add a string representation as tags.